Sources/XcodeProj/Objects/Files/PBXFileSystemSynchronizedRootGroup.swift

```diff
@@ -2,108 +2,113 @@ import Foundation
-    /// in the synchronized root group.
-    public var exceptions: [PBXFileSystemSynchronizedBuildFileExceptionSet]? {
-        set {
-            exceptionsReferences = newValue?.references()
-        }
-        get {
-            exceptionsReferences?.objects()
-        }
-    }
-
-    /// A list of relative paths to children folder whose configuration is overriden.
-    public var explicitFolders: [String]
-
-    /// Initializes the file element with its properties.
-    ///
-    /// - Parameters:
-    ///   - sourceTree: file source tree.
-    ///   - path: object relative path from `sourceTree`, if different than `name`.
-    ///   - name: object name.
-    ///   - includeInIndex: should the IDE index the object?
-    ///   - usesTabs: object uses tabs.
-    ///   - indentWidth: the number of positions to indent blocks of code
-    ///   - tabWidth: the visual width of tab characters
-    ///   - wrapsLines: should the IDE wrap lines when editing the object?
-    ///   - explicitFileTypes: It maps relative paths inside the synchronized root group to a particular file type.
-    ///   - exceptions: It returns a list of exception objects that override the configuration for some children in the synchronized root group.
-    ///   - explicitFolders: A list of relative paths to children folder whose configuration is overriden.
-    public init(sourceTree: PBXSourceTree? = nil,
-                path: String? = nil,
-                name: String? = nil,
-                includeInIndex: Bool? = nil,
-                usesTabs: Bool? = nil,
-                indentWidth: UInt? = nil,
-                tabWidth: UInt? = nil,
-                wrapsLines: Bool? = nil,
-                explicitFileTypes: [String: String] = [:],
-                exceptions: [PBXFileSystemSynchronizedBuildFileExceptionSet] = [],
-                explicitFolders: [String] = []) {
-        self.explicitFileTypes = explicitFileTypes
-        exceptionsReferences = exceptions.references()
-        self.explicitFolders = explicitFolders
-        super.init(sourceTree: sourceTree,
-                   path: path,
-                   name: name,
-                   includeInIndex: includeInIndex,
-                   usesTabs: usesTabs,
-                   indentWidth: indentWidth,
-                   tabWidth: tabWidth,
-                   wrapsLines: wrapsLines)
+  /// It maps relative paths inside the synchronized root group to a particular file type.
+  /// If a path doesn't have a particular file type specified, Xcode defaults to the default file type
+  /// based on the extension of the file.
+  public var explicitFileTypes: [String: String]?
+  
+  /// Returns the references of the exceptions.
+  var exceptionsReferences: [PBXObjectReference]?
+  
+  /// It returns a list of exception objects that override the configuration for some children
+  /// in the synchronized root group.
+  public var exceptions: [PBXFileSystemSynchronizedBuildFileExceptionSet]? {
```

Comment:
Is seems that the indentation is changed?

Reply:
Yeah. I wonder I just ran `swiftformat`. I'll check what happened.